bin/install/upload_to_s3.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -x
+
+COSMOS_VERSION="0.4.0"
+
+cosmos_cli()
+{
+        result=$(echo -e "$COSMOS_VERSION\n$TAG_VERSION" | sed '/^$/d' | sort -nr | head -1)
+        [[ "$result" = "$TAG_VERSION" ]]
+}
+
+if cosmos_cli ; then
+        aws s3 --region=us-east-1 cp \
+        %aws.bash_script_to_upload% \
+        %aws.bash_destination_url%
+
+        aws s3 --region=us-east-1 cp \
+        %aws.powershell_script_to_upload% \
+        %aws.powershell_destination_url%
+else
+        aws s3 --region=us-east-1 cp \
+        %aws.bash_script_to_upload% \
```

Comment:
Right but we need to upload a different file too?
On Tue, Feb 9, 2016 at 10:00 tamarrow <notifications@github.com> wrote:

> In bin/install/upload_to_s3.sh
> <https://github.com/mesosphere/dcos-cli/pull/448#discussion_r52347041>:
>
> > +{
> > +        result=$(echo -e "$COSMOS_VERSION\n$TAG_VERSION" | sed '/^$/d' | sort -nr | head -1)
> > +        [[ "$result" = "$TAG_VERSION" ]]
> > +}
> > +
> > +if cosmos_cli ; then
> > +        aws s3 --region=us-east-1 cp \
> > +        %aws.bash_script_to_upload% \
> > +        %aws.bash_destination_url%
> > +
> > +        aws s3 --region=us-east-1 cp \
> > +        %aws.powershell_script_to_upload% \
> > +        %aws.powershell_destination_url%
> > +else
> > +        aws s3 --region=us-east-1 cp \
> > +        %aws.bash_script_to_upload% \
>
> I'm using aws.bash_destination_legacy_url for the s3 location where
> non-cosmos cli releases will be uploaded (which will be in a different
> location than cosmos cli releases)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mesosphere/dcos-cli/pull/448/files#r52347041>.
>
-- 
José Armando García Sancio
Mesosphere